hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/MessageContentWrapperDispatcher.java

```diff
@@ -0,0 +1,44 @@
+package pl.allegro.tech.hermes.common.message.wrapper;
+
+import org.apache.avro.Schema;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.domain.topic.schema.SchemaRepository;
+
+import javax.inject.Inject;
+
+public class MessageContentWrapperDispatcher {
```

Comment:
this class does not dispatch any MessageContenetWrapper, why it isn't MessageContentWrapper?